### PR TITLE
[ID] Add `People/The_Team/Language_Surveillance_Division`

### PR DESCRIPTION
**Pull request by ZeroPyrozen**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/People/The_Team/Language_Surveillance_Division/id.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+LSD juga berfungsi menjadi grup perantara untuk member yang akan bergabung ke dalam [Global Moderation Team](/wiki/People/The_Team/Global_Moderation_Team) pada waktu itu, hal tersebut dikarenakan kedua grup tersebut memiliki tujuan dan membutuhkan anggota dengan kemampuan yang sama secara umum.
+
+## Restrukturisasi
+
+![](img/chat-moderator-profile.png "Halaman profil seorang chat moderator pada tahun 2013")
+
+Setelah [restrukturisasi staf](https://osu.ppy.sh/community/forums/topics/123510) pada tahun 2013, grup Language Surveillance Division diberi nama baru yaitu **Chat Moderator** dan memiliki keanggotaannya sendiri.
+
+<!--TODO: Translate Staff log-->
```
Comment (Walavouchey) [inline wiki/People/The_Team/Language_Surveillance_Division/id.md:28]: ?

Comment (ZeroPyrozen) [reply wiki/People/The_Team/Language_Surveillance_Division/id.md:28]: For next line

`Pada tanggal [20 Agustus 2015](/wiki/Staff_Log/2015#august)...`

File: wiki/People/The_Team/Language_Surveillance_Division/id.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  - chat operator
+  - chatops
+  - lsd
+---
+
+# Language Surveillance Division
+
+*Untuk artikel mengenai tim moderator modern, lihat: [Global Moderation Team](/wiki/People/The_Team/Global_Moderation_Team).*
+
+**Language Surveillance Division** (atau lebih dikenal sebagai *LSD*) merupakan grup pengguna yang bertanggungjawab untuk memoderasi komunitas-komunitas yang berbasis bahasa tertentu pada osu!. Setiap anggota memiliki area kerjanya sendiri, yaitu: sebuah *channel* bahasa yang sering mereka kunjungi dan sub-forum yang mereka atur (seperti sub-forum [Help](https://osu.ppy.sh/community/forums/5) atau [Off-Topic](https://osu.ppy.sh/community/forums/52)).
```
Comment (Nivalyx) [inline wiki/People/The_Team/Language_Surveillance_Division/id.md:14]:
```suggestion
**Language Surveillance Division** (atau yang lebih dikenal sebagai *LSD*) merupakan grup pengguna yang bertanggung jawab untuk memoderasi berbagai komunitas kebahasaan yang ada di osu!. Masing-masing anggota grup ini memiliki ruang tugasnya tersendiri, baik itu *channel* bahasa yang terkait atau sub-forum tertentu (seperti sub-forum [Help](https://osu.ppy.sh/community/forums/5) atau [Off-Topic](https://osu.ppy.sh/community/forums/52)) yang sering mereka kunjungi.
```

File: wiki/People/The_Team/Language_Surveillance_Division/id.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+## Tugas-tugas
```
Comment (Nivalyx) [inline wiki/People/The_Team/Language_Surveillance_Division/id.md:16]:
```suggestion
## Tugas
```

File: wiki/People/The_Team/Language_Surveillance_Division/id.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+  - chatop
+  - chat op
+  - chat operator
+  - chatops
+  - lsd
+---
+
+# Language Surveillance Division
+
+*Untuk artikel mengenai tim moderator modern, lihat: [Global Moderation Team](/wiki/People/The_Team/Global_Moderation_Team).*
```
Comment (Nivalyx) [inline wiki/People/The_Team/Language_Surveillance_Division/id.md:12]:
```suggestion
*Untuk pembahasan lebih lanjut seputar tim moderator modern yang saat ini aktif bertugas, lihat: [Global Moderation Team](/wiki/People/The_Team/Global_Moderation_Team).*
```

File: wiki/People/The_Team/Language_Surveillance_Division/id.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+
+## Tugas-tugas
+
+Anggota dari LSD dipilih secara internal oleh tim inti osu! dari komunitas untuk bertugas sebagai moderator pada komunitas dan sub-forum yang memiliki jumlah pengunjung yang rendah. Tidak seperti Global Moderator, mereka tidak memiliki fitur-fitur tertentu seperti *nickname* dengan warna khusus atau gelar khusus di forum. Namun di [IRC](/wiki/Community/Internet_Relay_Chat), mereka menjadi [operator](/wiki/Community/Internet_Relay_Chat#why-are-some-usernames-prefixed-with-different-signs?) dan dapat memberikan [silence](/wiki/Silence).
+
+LSD juga berfungsi menjadi grup perantara untuk member yang akan bergabung ke dalam [Global Moderation Team](/wiki/People/The_Team/Global_Moderation_Team) pada waktu itu, hal tersebut dikarenakan kedua grup tersebut memiliki tujuan dan membutuhkan anggota dengan kemampuan yang sama secara umum.
```
Comment (Nivalyx) [inline wiki/People/The_Team/Language_Surveillance_Division/id.md:20]:
```suggestion
LSD juga berfungsi sebagai grup perantara untuk para anggota yang akan bergabung ke dalam [Global Moderation Team](/wiki/People/The_Team/Global_Moderation_Team) pada masanya. Hal ini dikarenakan kedua grup tersebut secara umum memiliki tujuan yang serupa dan membutuhkan anggota dengan kemampuan yang kurang lebihnya sama.
```

File: wiki/People/The_Team/Language_Surveillance_Division/id.md [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+
+## Restrukturisasi
+
+![](img/chat-moderator-profile.png "Halaman profil seorang chat moderator pada tahun 2013")
```
Comment (Nivalyx) [inline wiki/People/The_Team/Language_Surveillance_Division/id.md:24]:
```suggestion
![](img/chat-moderator-profile.png "Profil salah seorang anggota Chat Moderator pada tahun 2013")
```

File: wiki/People/The_Team/Language_Surveillance_Division/id.md [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+
+Setelah [restrukturisasi staf](https://osu.ppy.sh/community/forums/topics/123510) pada tahun 2013, grup Language Surveillance Division diberi nama baru yaitu **Chat Moderator** dan memiliki keanggotaannya sendiri.
```
Comment (Nivalyx) [inline wiki/People/The_Team/Language_Surveillance_Division/id.md:26]:
```suggestion
Seiring dengan [restrukturisasi staf](https://osu.ppy.sh/community/forums/topics/123510) yang berlangsung pada tahun 2013, grup Language Surveillance Division dialihfungsikan menjadi **Chat Moderator** yang memiliki status keanggotaannya tersendiri.
```

File: wiki/People/The_Team/Language_Surveillance_Division/id.md [hunk 8]
```diff
@@ -0,0 +1,30 @@
...
+
+<!--TODO: Translate Staff log-->
+
+Pada tanggal [20 Agustus 2015](/wiki/Staff_Log/2015#august) Chat Moderator dan Global Moderation Team [dimerger](https://osu.ppy.sh/community/forums/posts/4435972) dan membuat seluruh moderator memiliki warna merah pada namanya dan tingkat akses manajemen untuk seluruh forum.
```
Comment (Nivalyx) [inline wiki/People/The_Team/Language_Surveillance_Division/id.md:30]:
```suggestion
Pada tanggal [20 Agustus 2015](/wiki/Staff_Log/2015#august), grup Chat Moderator dan Global Moderation Team [dilebur menjadi satu](https://osu.ppy.sh/community/forums/posts/4435972) yang secara tidak langsung menyetarakan nama dan hak moderasi yang dimiliki oleh seluruh moderator.
```

File: wiki/People/The_Team/Language_Surveillance_Division/id.md (hunk 1 above)
Comment (Walavouchey) [reply wiki/People/The_Team/Language_Surveillance_Division/id.md:28]: but does this note need to be here? unless some changes in this file depend on a future translation, the comment doesn't really do much

Comment (ZeroPyrozen) [reply wiki/People/The_Team/Language_Surveillance_Division/id.md:28]: Okay let me remove that

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**